src/style.css

```diff
@@ -0,0 +1,26 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
don't reset indentation for the *

Reply:
Can you explain to me why I cannot use * for resetting styles?

Reply:
because * is selecting everything, it's bad practice to reset styles so